packages/web/app/src/index.css

```diff
@@ -203,3 +203,7 @@
 .hive-badge-is-changed:after {
   @apply absolute right-2 size-1.5 rounded-full border border-orange-600 bg-orange-400 content-[''];
 }
+
+#root > * {
+  background-color: rgb(3 7 17 / var(--tw-bg-opacity, 1)) !important;
+}
```

Comment:
This is a global style that is applied to body, but somehow does not cascade to the react app root containers children...